package.json

```diff
@@ -28,9 +28,6 @@
     "email": "ismaelga@gmail.com"
   },
   "main": "dist/react-json-editor.js",
-  "peerDependencies": {
-    "react": ">=0.12.2 <1.0.0"
-  },
```

Comment:
I think we should go with `"react": "^0.14.0 || ^15.0.0"` instead of removing it completely.